Replicator/Pusher+DB.cc

```diff
@@ -209,7 +209,7 @@ namespace litecore { namespace repl {
             }
 
             if (_options.pushFilter) {
-                if (!_options.pushFilter(doc->docID, doc->selectedRev.flags,
+                if (!_options.pushFilter(doc->docID, doc->revID, doc->selectedRev.flags,
```

Comment:
should be `doc->selectedRev.revID`

Reply:
What is the difference?  I thought this was shorthand for that.